FirebasePerformance/Sources/AppActivity/FPRSessionDetails.h

```diff
@@ -31,7 +31,7 @@ typedef NS_OPTIONS(NSUInteger, FPRSessionOptions) {
 @property(nonatomic, readonly) FPRSessionOptions options;
 
 /* Length of the session in minutes. */
-@property(nonatomic, readonly) NSUInteger sessionLengthInMinutes;
+- (NSUInteger)sessionLengthInMinutes:(nonnull NSDate *)now;
```

Comment:
Since we are passing parameter now, recommend naming this API: `sessionLengthInMinutesFromDate:(NSDate *) now`.

Reply:
Done